src/xcode/ENA/ENA/Source/Scenes/SelectValueViewController/SelectValueTableViewController.swift

```diff
@@ -44,7 +44,8 @@ final class SelectValueTableViewController: UITableViewController {
 
 	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
 		let cell = tableView.dequeueReusableCell(cellType: SelectValueTableViewCell.self, for: indexPath)
-		cell.configure(viewModel.cellViewModel(for: indexPath))
+		let isEnabled = indexPath.item == 0 ? viewModel.isInitialCellEnabled : true
```

Comment:
Just a minor detail but something I find hard to read.
Normally ternary operator works as follows: `<condition> ? <true case> : <false case>`. Having the `true` value in the `false case` is somewhat odd. How about:
```suggestion
		let isEnabled = indexPath.item > 0 ? true : viewModel.isInitialCellEnabled
```
If you are really paranoid and expect bugs in `IndexPath` try: `!= 0` instead.